### ISSUE TITLE
[ask] why who don't have a license key can only send 15mb video/audio?

### ISSUE THREAD

**Issue by Sansekai** [open-wa/wa-automate-nodejs#2806]
I have tried both.

those who have a license key can send 50 mb audio/video and those who don't have it can only send 15 mb audio/video.

**Comment by smashah** [comment]
I have just checked the code.

The code to unlock max file size in for licensed users is there. When I implemented it, it did not make any difference, hence no documentation.

There is NO CODE THAT LIMITS UPLOAD size for unlicensed accounts. It is most likely some silly WA update changed the upload limit which in turn made it so the licensed code now actually makes a difference.

I was 99% sure I didn't limit the upload size, but I don't recall everything at all times so instead of denial I wanted to check. I have checked. There is no artificial limitation from open-wa. By coincidence the dormant code was made to be effective.

I will be migrating the licensed code to patches. This should eliminate any difference in upload size between licensed and unlicensed sessions.
